common/lib/xmodule/xmodule/course_module.py

```diff
@@ -663,7 +663,14 @@ class CourseFields(object):
         # Ensure that courses imported from XML keep their image
         default="images_course_image.jpg"
     )
-
+    issue_badges = Boolean(
+        display_name=_("Issue OpenBadges"),
```

Comment:
@catong (I'm a developer on Badgr, the application that this integration works with): "Open Badges" is the name of an open, free-to-use technical specification for describing achievements, which is implemented by many pieces of software. It's not specific to the Badgr platform. There are digital badges that are not Open Badges (meaning they can't be taken out of the system in which they were issued), but non-open badges would not work with other parts of this integration. :+1: to keeping your eyes open to places where you could swap out one dependency for another, but in this case, even if you replaced Badgr with a fork or different compatible software, you would still be issuing Open Badges. I would say that there is usually a space between "Open Badges". See http://openbadges.org and https://openbadgespec.org